subsys/caf/modules/power_manager.c

```diff
@@ -76,6 +85,42 @@ static void power_down_counter_abort(void)
 	LOG_DBG("Power down timer aborted");
 }
 
+#if IS_ENABLED(CONFIG_CAF_POWER_MANAGER_RUNTIME_TIMEOUT_SET)
+static int settings_set(const char *key, size_t len_rd, settings_read_cb read_cb, void *cb_arg)
+{
+	if (!strcmp(key, POWER_DOWN_TIMEOUT_CONFIG_KEY)) {
+		ssize_t len = read_cb(cb_arg, &power_down_timeout, sizeof(int));
+
+		if ((len != sizeof(power_down_timeout)) || (len != len_rd)) {
+			LOG_ERR(
```

Comment:
please keep the argument at the same line as function name, and break later on. that is:

```
   fun_name(some_arguments, other_args,
                    args_that_crossed_the_line);
```

fix in all places

Reply:
still pending